src/common/tiles/MovieTile/index.js

```diff
@@ -19,21 +20,40 @@ export const MovieTile = ({
   rating,
   votes,
   genre_ids,
+  character,
+  job,
+  year,
 }) => {
   return (
-    <Container key={id} id={id}>
-      <Poster
-        src={poster_path ? `${imageBaseUrl}/w342${poster_path}` : noPoster}
-        alt={`poster of ${title}`}
-      ></Poster>
-      <InfoContent>
-        <Details>
-          <Title>{title}</Title>
-          <Year>{release_date}</Year>
-          <Genre genre_ids={genre_ids} />
-        </Details>
-        <StyledRatings rating={rating} votes={votes} />
-      </InfoContent>
-    </Container>
+    <MovieStyledLink to={`/movie-page/${id}`} key={id}>
+      <Container>
+        <Poster
+          src={poster_path ? `${imageBaseUrl}/w342${poster_path}` : noPoster}
+          alt={`poster of ${title}`}
+        ></Poster>
+        <InfoContent>
+          <Details>
+            {title && <Title>{title}</Title>}
+
+            {release_date && <Year>{release_date}</Year>}
+
+            {character && (
+              <Year>
+                {character} {year}
+              </Year>
+            )}
+
+            {job && (
+              <Year>
+                {job} {year}
+              </Year>
+            )}
```

Comment:
Trochę nie rozumiem. Co to ma robć?

Reply:
StyledComponent Year ma sie wyswietlac tylko jesli mamy dany element przekazany przez referencje, job i character to pola z odpowiednio cast i crew dlatego sa zapisane osobno

Reply:
Kurczę, dalej tego nie czuję. Pogadamy na callu :D